scss/_patterns_strip.scss

```diff
@@ -1,17 +1,37 @@
 @import 'settings';
 
-@mixin vf-strip {
-  clear: both;
-  margin-top: 0;
-  padding: $sp-x-large 0;
-  width: 100%;
-
-  @media only screen and (min-width: $breakpoint-large) {
-    padding: 4rem 0;
-  }
-}
 
 @mixin vf-p-strip {
+  %section-padding--default--max-bp-large {
+    @media only screen and (max-width: $breakpoint-large) {
+      padding-bottom: $spv-inter--scaleable;
+      padding-top: $spv-inter--scaleable;
+    }
+  }
+
+  %section-padding--default--min-bp-large {
+    @media only screen and (min-width: $breakpoint-large) {
+      padding-bottom: $spv-inter--scaleable * 2;
+      padding-top: $spv-inter--scaleable * 2 - $px; //border compensation
+    }
+  }
+
+  %section-padding--default {
+    @extend %section-padding--default--max-bp-large;
+    @extend %section-padding--default--min-bp-large;
```

Comment:
We dont need placeholders for styles we use once.

Reply:
done